platform/country_defines.hpp

```diff
@@ -8,7 +8,8 @@ enum class MapOptions : uint8_t
   Nothing = 0x0,
   Map = 0x1,
   CarRouting = 0x2,
-  MapWithCarRouting = 0x3
+  MapWithCarRouting = 0x3,
```

Comment:
It should be removed before start all this changes with Storage. But the proposal was rejected by @burivuh because of a lack of time. I said about it two weeks ago. He promised to find time for this refactoring at next iteration.